neo4j-quickstart/src/test/java/org/acme/neo4j/FruitsEndpointTest.java

```diff
@@ -0,0 +1,71 @@
+package org.acme.neo4j;
+
+import io.quarkus.test.junit.QuarkusTest;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.UnaryOperator;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+
+import org.junit.jupiter.api.Test;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.core.IsNot.not;
+
+@QuarkusTest
+class FruitsEndpointTest  {
+
+    private static final Map<String, String> fruitIdMap = new HashMap<>();
+    private static final UnaryOperator<String> FRUIT = fruit ->  String.format("{\"name\":\"%s\"}", fruit);
+
+    @Test
+    public void testResource() {
```

Comment:
Would be great to also test the reactive API if possible.

Reply:
I use it in line 60. Would it be better to create other test class? It doesn't have a persist method, so I did it in the same one

Reply:
sorry, I didn't see it! that's great!